cfn-templates/cid-cfn.yml

```diff
@@ -16,7 +16,7 @@ Metadata:
           - CURBucketPath
           - CURIsManagedByCloudFormation
       - Label:
-          default: Available dashboards. Keep all 'no' for the first deployment. Go to QuickSight, change Permission and after that update the stack putting to 'yes' 3 first dashboards.
+          default: Available dashboards. CUDOS CostIntelligenceDashboard and KPI are CUR based, others requre installation of an additional Lab.
```

Comment:
I'd suggest to re-phrase to 'Trusted Advisor and Compute Optimiser Dashboards require deployment of Optimization Data Collection Lab (https://wellarchitectedlabs.com/cost/300_labs/300_optimization_data_collection/)